memory/hugepage_sanity.py.data/README.md

```diff
@@ -0,0 +1,20 @@
+Overview
+----------
+The hugepage_sanity test, tests the allocation of hugepages on Linux systems. It allocates a specified amount hugepages and then performs write operations to verify that the memory is accessible and functioning correctly. It primarily uses mmap system call and memset function to allocate the hugepages.
+
+
+Prerequisites
+--------------
+This test script requires a Linux system with hugepages support enabled. You must have the hugetlbfs supported in order to use hugepages.
+Additionally, you will need Python 3.x and the avocado-framework package installed on the Linux system.
+
+
+Parameters
+--------------
+* hpagesize : the size of hugepage in MB you want to run the test on, by default it is the default size supported on the kernel
+Example : hpagesize: 2
+This runs the test for hugepage size of 2MB
+
+* num_pages : number of hugepages that the test should allocate, by default it is 1
+Example : num_pages: 2
```

Comment:
same here, the range it supports

Reply:
I have addressed this and comment above by adding :
`Make sure the number is such that there is sufficient memory available with the system to function properly.`

there is no such thing as a proper range it is system dependent.